integration/gobash.go

```diff
@@ -0,0 +1,116 @@
+// +build integration
```

Comment:
can you put this and `process.go` in a separate package?  `integration/exec` or something.